base/base/crc32_s390x.h

```diff
@@ -0,0 +1,26 @@
+#pragma once
```

Comment:
Please let me know if this file's name is okay. More than happy to change it if requested 😁

Reply:
Let's pay attention to disambiguating CRC32 and CRC32-C, because it is a common source of confusion. If this file implements CRC32-C, name it accordingly. If it implements both, it is ok to use CRC32 without additional C in the name.

Reply:
Thank you for pointing that out. After some testing, CRC32-C is certainly all that we are interested in here. I've updated the name of the file, as well as updated the functions name to better reflect what is being used.